Sources/BuilderIO/Helpers/RenderContent.swift

```diff
@@ -8,31 +8,48 @@ public struct RenderContent: View {
         self.content = content
         if (!RenderContent.registered) {
             // TODO: move these out of here?
+            var builderIds = [String]()
             registerComponent(name: "Text", factory: { options in
-                print("Text \n titleString = "+options["text"].stringValue+" \n options = \(options)")
+                // print("Text \n titleString = "+options["text"].stringValue+" \n options = \(options)")
                 return BuilderText(text: options["text"].stringValue)
             })
             registerComponent(name: "Image", factory: { options in
                 return BuilderImage(image: options["image"].stringValue, backgroundSize: options["backgroundSize: <#T##String#>"].stringValue)
             })
             registerComponent(name: "Core:Button", factory: { options in
+                // print("Core:Button \n titleString = "+options["text"].stringValue+" \n options = \(options) \n content.data = \(content.data)")
                 for block in content.data.blocks {
-                    if(options["text"] == block.component?.options?["text"]) {
+                    if(options["text"] == block.component?.options?["text"] && !builderIds.contains(block.id)) {
```

Comment:
this seems incomplete? Where do we add to builderIds? Also, in general, that still doesn't solve for the problem i think

Reply:
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/116343259/204367869-d47357a6-a036-4999-a886-e9428b4cfc2c.png">

I will make the above change and will submit it in the next commit.

Please let me know if you have anything on this.